src/ADMM.jl

```diff
@@ -230,7 +235,7 @@ function iterate(solver::ADMM{matT,opT,T,preconT}, iteration::Int=0) where {matT
     copyto!(solver.zᵒˡᵈ[i], solver.z[i])
     solver.z[i][:] .= solver.regTrafo[i]*solver.x .+ solver.u[i]
     if solver.ρ[i] != 0
-      solver.reg[i].prox!(solver.z[i], solver.regFac*solver.reg[i].λ/solver.ρ[i]; solver.reg[i].params...)
+      solver.z[i] = solver.reg[i].prox!(solver.z[i], solver.regFac*solver.reg[i].λ/solver.ρ[i]; solver.reg[i].params...)
```

Comment:
Why is this assigned back to `solver.z[i]`? `prox!` has the mutating sign indicating that `solver.z[i]` is changed.

Reply:
That was to circumvent a bug I had introduced myself in `proxLLR!`. I fixed the bug and removed the assignment.